.travis.yml

```diff
@@ -1,33 +1,66 @@
 language: cpp
 sudo: false
-compiler: clang
-#    - clang
-#    - gcc
+compiler:
+    - clang
+    - gcc
 
-os: osx
-#    - linux
-#    - osx
+os:
+    - linux
+    - osx
+#osx_image: xcode7
+dist: trusty
+
+addons:
+  apt: 
+   sources:
+      - boost-latest
+      - ubuntu-toolchain-r-test
+   packages:
+      - gcc-4.8
+      - g++-4.8
+      - libboost1.55-all-dev
+      - libilmbase-dev
+      - libopenexr-dev
+      - libgif-dev
+      - libtiff4-dev
+      - libopenjpeg-dev
+      - libwebp-dev
+      - ffmpeg
```

Comment:
The install_homebrew_deps.bash is OSX only.

This bit here here is Linux only.

The "apt:" category is for a Linux-side plugin that runs apt-get on the packages you list. You have to do it that way if you use "containers", which is encouraged, because containers don't let you 'sudo' which is ordinarily required for apt-get. Sources means servers to apt-get from. Packages is which packages to install.